### ISSUE TITLE
fix 4k vnc on the TV

### ISSUE THREAD

**Issue by ja5087** [ocf/puppet#894]
Now that the TV is 4k, xtigervncviewer will show scroll bars which is not nice. While it supports resizing, this is only if the server also supports it which x11vnc does not. We could either:

1) Switch to a viewer that supports client-side resizing such as Vinagre or KRDC or TightVNC. Vinagre is a little heavy on console output but we can filter it out. KRDC has a lot of dependencies. 
2) Switch the VNC server to TigerVNC.

I wanted to get some discussion before submitting a PR.

**Comment by cg505** [comment]
Switching the VNC server may also fix the issues with it randomly crashing.

**Comment by ethanhs** [comment]
Yeah I think switching the server seems like the best path forward, are there any downsides?

**Comment by jvperrin** [comment]
I struggled to find a good VNC server setup when doing this and tried like 3-4 different packages before settling on `x11vnc`, but that was also a while ago so the landscape may have changed since then. Hopefully it's easier to find a good working setup now
